helm/values.yaml

```diff
@@ -33,6 +33,9 @@ coder:
     # It is recommended to keep this on if you are using Kubernetes templates
     # within Coder.
     workspacePerms: true
+    # coder.serviceAccount.annotations -- The Coder service account annotations.
+    annotations:
+      iam.gke.io/gke-metadata-server-enabled: "true"
```

Comment:
I'm not sure if we should set this by default as it may confuse non-GKE users.

Reply:
ah apologies - this mistakenly got committed. was using it as a test annotation.